externs/typeahead.js

```diff
@@ -151,22 +151,23 @@ var TypeaheadDataset;
  * @typedef {{
  *   highlight: (boolean|undefined),
  *   hint: (boolean|undefined),
- *   minLength: (number|undefined)
+ *   minLength: (number|undefined),
+ *   classNames: (Object|undefined)
  * }}
  */
 var TypeaheadOptions;
 
 /**
- * @param {string|TypeaheadOptions} options
- * @param {...TypeaheadDataset} var_dataset
+ * @param {TypeaheadOptions|string} options
+ * @param {...TypeaheadDataset|string} var_dataset
  * @return {!jQuery}
  */
 jQuery.prototype.typeahead = function(options, var_dataset) {};
 
 
 /**
- * @param {string|TypeaheadOptions} options
- * @param {...TypeaheadDataset} var_dataset
- * @return {!angular.JQLite}
+ * @param {TypeaheadOptions|string} options
+ * @param {...TypeaheadDataset|string} var_dataset
+ * @return {!jQuery}
```

Comment:
Are you sure about this change (from `angular.JQLite` to `jQuery`)? Note that there is `jQuery.prototype.typeahead` when jQuery is used.